.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    if: "!contains(github.event.head_commit.message, 'ci skip') && !contains(github.event.head_commit.message, 'skip ci')"
```

Comment:
Is this recommended by the auto people? It would be better to have just one special code phrase to skip this workflow. But then again, I don't really understand what this is for. Maybe to prevent some kind of horrific recursive infinite release loop?

Reply:
> Maybe to prevent some kind of horrific recursive infinite release loop?

Yup - https://intuit.github.io/auto/docs/build-platforms/github-actions

Reply:
Weird that they have two trigger phrases 🤷🏾